api/routes/users.js

```diff
@@ -203,6 +203,42 @@ router.get('/picture', async function (req, res) {
     }
 });
 
+router.post('/modifyfavorites/:operation', async function(req, res){
+    let userId = req.body.userId;
+    let gameList = req.body.gameList;
+    let operation = req.params.operation;
+    let objId = new ObjectId(userId);
+    console.log(operation);
```

Comment:
remove this, only really log errors or weird edge cases

Reply:
removed the console.log